src/webserver_main.cpp

```diff
@@ -19,7 +19,9 @@
 #include <obmc_shell.hpp>
 #include <openbmc_dbus_rest.hpp>
 #include <redfish.hpp>
+#ifdef BMCWEB_ENABLE_REDFISH_AGGREGATION
```

Comment:
We moved away from doing this for the includes 
https://github.com/openbmc/bmcweb/commit/22db1728d5fdcac0a257976afa99034ae189c753